### ISSUE TITLE
Load a subdeck

### ISSUE THREAD

**Issue by Qriist** [Cockatrice/Cockatrice#2562]
I am requesting the ability for the host (or all players, potentially) to load a subdeck at the normal deck loading screen. This is important for some formats such as Planechase and Archenemy, where the cards cannot be part of a player deck but are NEEDED for the fomat.

Perhaps a small amount of the stack space (at the bottom, where cards rarely get to, visually) can be allocated to tracking, interacting and accessing this zone.

**Comment by ZeldaZach** [comment]
I fully agree with the idea of having mutual space, but it's not feasible at this time. We hope to look into this in the future.

**Comment by Daenyth** [comment]
We likely won't be doing this as "subdeck" but instead a different game mode.

**Comment by Qriist** [comment]
I would like to clarify that I mean loading two decks simultaneously while in the game lobby.

While different game modes would be nice (Momir Vig, anyone?) I think the subdeck option is really the way to go. A lot of times we MTG folk mix'n'match modes and it would just be much more convenient to have only one screen to reference. Can you possibly think of all the variations on a mode?

@ZeldaZach , I wasn't asking for mutual space, just pointing out a location where the given person's secondary deck could be accessed on screen, by that person. Let's say the host loads a deck of schemes for archenemy; it would be more user-friendly for the host to be able to drag them off the top there in the game than to have to flip back and forth between screens.



Hope all of that is slightly less confusing.

**Comment by Daenyth** [comment]
Shared deck zone layout would be an alternate mode. It would let you do
mental magic, Ascension, Rochester draft, Winston draft, Dominion, many
others

**Comment by ZeldaZach** [comment]
Linking #3416 and #1740
